index.html

```diff
@@ -222,5 +222,37 @@ <h2>Skills</h2>
             </div>
         </div>
     </section>
+    <section class="contact-form" id="contact">
+        <div class="contact-text">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <div class="contact-area">
+            <form action="https://formspree.io/f/xvolllzo" method="post" id="frmContact">
+              <label for="name">
+                <input type="text" name="name" id="name" maxlength="30" class="contact-input" placeholder="Enter your names..."
+                required/>
+              </label>
+              <label for="email">
+                <input type="email" name="email" id="email" class="contact-input" placeholder="Enter your gmail..."
+                required/>
+              </label>
+              <label for="text">
+                <textarea name="text" id="text" maxlength="500" class="contact-input" placeholder="Enter text here" cols="23" rows="5"></textarea>
+              </label>
+                <input id="btn" type="submit" class="submit-btn dark-font" value="Get in Touch" />
```

Comment:
- [ ] You have done a tremendous job styling your submit button. However, so that you stick as much as possible to the chosen Figma design, kindly make sure that you implement the button states eg on hover, on submit etc.This is the missing piece to perfection.

**Below are the desired button states**
![image](https://user-images.githubusercontent.com/86318284/177408159-f9cf0cb1-506d-4d47-883b-f8b134b73d97.png)

**It would also be a good thing to have the `cursor: pointer;` CSS rule to make it intuitive when your users try to submit  on your form**

- [ ]  Also, kindly maintain the project's Figma design in terms of colors, fonts, etc. Your input button currently does not meet all the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Kindly fix this.

**This is how your project currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/177408543-9e1f4754-303e-4a37-9a68-9698488826d4.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/177408670-d3fcfc4d-750d-4e16-b8e2-a606e25eb692.png)